springql-core/src/http_blocking.rs

```diff
@@ -0,0 +1,83 @@
+// This file is part of https://github.com/SpringQL/SpringQL which is licensed under MIT OR Apache-2.0. See file LICENSE-MIT or LICENSE-APACHE for full license details.
+
+use std::time::Duration;
+
+use reqwest::StatusCode;
+use serde_json::Value;
+
+#[derive(Debug)]
+pub struct ReqwestClient(reqwest::blocking::Client);
+#[derive(Debug)]
+pub struct Response(reqwest::blocking::Response);
+
+#[derive(Debug)]
+pub struct ReqwestError(reqwest::Error);
+
+#[derive(Debug)]
+pub struct RequestBuilder {
```

Comment:
I feel `Request` would be a better name because it does not build a request but `.send()` an HTTP request.

Reply:
this struct wraps `RequesrBuilder` come from reqwest, and rust defact standard `http` crate

Reply:
Wow, reqwest::RequestBuilder actually has `.send()` 😮 
https://docs.rs/reqwest/latest/reqwest/struct.RequestBuilder.html#method.send